simplecpp.cpp

```diff
@@ -2842,6 +2842,14 @@ static bool preprocessToken(simplecpp::TokenList &output, const simplecpp::Token
     return true;
 }
 
+static void addMaybeUsedMacro(std::map<std::string, std::list<simplecpp::Location> > &macros, const simplecpp::Token *tok) {
+    if(tok == nullptr) return;
+    if(macros.count(tok->str()) == 0) {
```

Comment:
as far as I see you can replace this:
```
    if(macros.count(tok->str()) == 0) {
        macros.insert(std::make_pair(tok->str(), std::list<simplecpp::Location>()));
    }
    macros.find(tok->str())->second.push_back(tok->location);
```
with:
```
    macros[tok->str()].push_back(tok->location);
```

Reply:
Sorry, once again: not really familiar with C++. Well, if this is the case, not really that worth it having the helper function. Let me refactor this.